src/graph/ShowExecutor.cpp

```diff
@@ -79,6 +81,12 @@ void ShowExecutor::execute() {
         case ShowSentence::ShowType::kShowSnapshots:
             showSnapshots();
             break;
+        case ShowSentence::ShowType::kShowCharset:
```

Comment:
How about show them in `show spaces`?

Reply:
show spaces  not display charset,  only show create space XXX to display space charset.